api/tests/integration/infrastructure/repositories/prescriber-repository_test.js

```diff
@@ -146,6 +146,58 @@ describe('Integration | Infrastructure | Repository | Prescriber', function () {
         );
       });
 
+      describe('#participantCount', function () {
```

Comment:
**suggestion** : rajouter un test `should not count disabled organization learners`